src/vs/workbench/electron-browser/window.ts

```diff
@@ -418,21 +417,15 @@ export class NativeWindow extends Disposable {
 		// Touchbar menu (if enabled)
 		this.updateTouchbarMenu();
 
-		// Crash reporter (if enabled)
-		if (!this.environmentService.disableCrashReporter && this.configurationService.getValue('telemetry.enableCrashReporter')) {
-			const companyName = this.productService.crashReporter?.companyName || 'Microsoft';
-			const productName = this.productService.crashReporter?.productName || this.productService.nameShort;
-
-			// With a provided crash reporter directory, crashes
-			// will be stored only locally in that folder
-			if (this.environmentService.crashReporterDirectory) {
-				this.setupCrashReporter(companyName, productName, undefined, this.environmentService.crashReporterDirectory);
-			}
-
-			// With appCenter enabled, crashes will be uploaded
-			else if (this.productService.appCenter) {
-				this.setupCrashReporter(companyName, productName, this.productService.appCenter, undefined);
-			}
+		// Add extra parameters to crash reporter (if enabled)
+		if (!this.environmentService.disableCrashReporter) {
```

Comment:
Is this a no-op if the crash reporter is not started? We do only seem to be checking on the CLI wether crash reporter is disabled, but not `argv.json`